cmd/server/config.go

```diff
@@ -6,22 +6,29 @@ import (
 	"os"
 
 	"github.com/ava-labs/avalanche-rosetta/service"
+	ethcommon "github.com/ethereum/go-ethereum/common"
 )
 
 var (
 	errMissingRPC           = errors.New("avalanche rpc endpoint is not provided")
 	errInvalidMode          = errors.New("invalid rosetta mode")
 	errGenesisBlockRequired = errors.New("genesis block hash is not provided")
+	errInvalidTokenAddress  = errors.New("invalid token address provided")
+	errInvalidIngestionMode = errors.New("invalid rosetta ingestion mode")
 )
 
 type config struct {
-	Mode             string `json:"mode"`
-	RPCEndpoint      string `json:"rpc_endpoint"`
-	ListenAddr       string `json:"listen_addr"`
-	NetworkName      string `json:"network_name"`
-	ChainID          int64  `json:"chain_id"`
-	LogRequests      bool   `json:"log_requests"`
-	GenesisBlockHash string `json:"genesis_block_hash"`
+	Mode                       string   `json:"mode"`
+	RPCEndpoint                string   `json:"rpc_endpoint"`
+	ListenAddr                 string   `json:"listen_addr"`
+	NetworkName                string   `json:"network_name"`
+	ChainID                    int64    `json:"chain_id"`
+	LogRequests                bool     `json:"log_requests"`
+	GenesisBlockHash           string   `json:"genesis_block_hash"`
+	IngestionMode              string   `json:"ingestion_mode"`
+	StandardModeTokenWhitelist []string `json:"standard_token_addresses"`
+
+	IndexDefaultTokens bool `json:"index_default_tokens"`
```

Comment:
I recommend using more informative naming here. I think this is `IndexUnknownTokens`?